### ISSUE TITLE
Electron node-gyp canvas package not found.

### ISSUE THREAD

**Issue by paragdiwan** [webhintio/webhint.io#942]

**Comment by molant** [comment]
Hi @paragdiwan,

Sorry for the late response. 
From the error it seems like you are running it directly inside electron. Is that right? Native modules need to be recompiled for Electron and unfortunately `canvas` is not an easy one to compile...

What is the scenario you are trying to achieve? `canvas` is used by the jsdom connector but you can probably use the puppeteer connector.

**Comment by paragdiwan** [comment]
> Hi @paragdiwan,
> 
> Sorry for the late response.
> From the error it seems like you are running it directly inside electron. Is that right? Native modules need to be recompiled for Electron and unfortunately `canvas` is not an easy one to compile...
> 
> What is the scenario you are trying to achieve? `canvas` is used by the jsdom connector but you can probably use the puppeteer connector.

@molant 
Yes you said right. I am trying to run `webhints` within the electron. irrespective of any connector, i.e. even before the execution step, the electron `rebuilding` process gives the error.

**Comment by molant** [comment]
Do you have a repo we can clone with some steps to reproduce the error?
I'm not sure what do I have to do to see the same problem you do right now.

Thanks!

**Comment by hxlnt** [comment]
@paragdiwan, just following up to see if you have a sample repo you could share so we can try to repro your issue. Thanks!
